wuvt/templates/base.html

```diff
@@ -35,24 +35,24 @@
     <div id="mainheader">
         <h1>
             <a href="{{ url_for('blog.index') }}" title="{{ config.STATION_NAME }}">
-                WUVT: Radio for Everyone! 90.7 FM
-            </a>
-        </h1>
+                <img id="logo" src="{{ url_for('static', filename='img/logo.svg') }}" alt="WUVT: Radio for Everyone! 90.7 FM" />
+	    </a>
+	<img id="hamburger" src="{{ url_for('static', filename='img/Hamburger_icon.svg') }}" />
+	</h1>
 
         <div id="robot">
         </div>
-    </div>
-
-    <div id="bubble">
+	<div id="bubble">
         {% block bubble %}
         <ul>
-            <li><a href="{{ url_for('page', slug='listen-live') }}">Listen Live!</a></li>
+            <li id="listen-live"><a href="{{ url_for('page', slug='listen-live') }}">Listen Live!</a></li>
```

Comment:
please add IDs for all of these, in case we need to tweak styling in the future.